packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.Props.ts

```diff
@@ -34,6 +34,9 @@ export interface ISwatchColorPickerProps {
    */
   updateButtonIconWithColor?: boolean;
 
+  /**
+   * The items to render the contents of the swatch color picker
```

Comment:
Can this comment be rephrased? Perhaps "the color options that will be made available to the user" or the "items corresponding to the available color choices"?